prov/usnic/Makefile.include

```diff
@@ -19,6 +19,7 @@ libusnic_direct_sources = \
 	prov/usnic/src/usnic_direct/usd_device.c \
 	prov/usnic/src/usnic_direct/usd_device.h \
 	prov/usnic/src/usnic_direct/usd_event.c \
+	prov/usnic/src/usnic_direct/usd_enum.c \
```

Comment:
Did you add this file?  I don't see it in this PR.  Or is this a separate / different fix?

Also remember that USD canonical source is SVN (sigh).